elastic/eui
# Upgrade to Yarn Berry (v4)

## Summary

Closes #7794.

This PR configures Yarn Berry / Yarn v4 to be used in EUI instead of Yarn Classic.

## QA

1. [ ] Checkout this PR locally (`gh pr checkout 7810`), run `yarn`, and ensure the development server (`yarn start`) works, and tests (`yarn test`) pass locally

### General checklist

- Browser QA
- Docs site QA
